templates/openstack-userdata.sh.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+# Write .env contents
+su - ubuntu -c 'cd ~/actions-runner && echo "{{ env_contents }}" > .env'
+
+# Create the runner and start the configuration experience
+su - ubuntu -c "cd ~/actions-runner && ./config.sh \
+    --url {{ github_url }} \
+    --token {{ token }} --ephemeral --unattended \
+    --labels {{ instance_labels }} --name {{ instance_name }}"
+
+# Run runner
+su - ubuntu -c "cd ~/actions-runner && /home/ubuntu/actions-runner/run.sh"
+
+# cleanup
+sleep 2
```

Comment:
Line 17 - 19 does not seems to be cleanup.
Is there a reason for 2 seconds? Are we waiting for something?

Reply:
Removed, thank you!
(This was a placeholder for post-runner actions)